Packs/MISP/Integrations/MISPV3/MISPV3.py

```diff
@@ -1662,7 +1664,7 @@ def delete_attribute_command(demisto_args: dict) -> CommandResults:
     """
     Gets an attribute id and deletes it.
     """
-    attribute_id = demisto_args.get('attribute_id')
+    attribute_id = demisto_args['attribute_id']
```

Comment:
same issue

Reply:
@shmuel44 if you verified each and every arg we changed `.get` to`[]` is required, I'm ok with that.